### DISCUSSION TITLE
How to change rule for application from GUI?

### DISCUSSION THREAD

**Discussion by junglized** [evilsocket/opensnitch#990] [Q&A] [answered]
Hi all,

I chose deny forever for an app by mistake. Is there a way to change that from the GUI. I feel like I cklicked everywhere alredy in the GUI and can't find relevant button/switch to bring up a menu where I can change that.

Thanks for all help.

**Comment by junglized** [comment, marked as answer]
Silly me, I've found it in Rules screen. Sorry for taking your time.

**Reply by gustavo-iniguez-goya** [reply]
no problem @junglized ! :)
